tools/testbuild.sh

```diff
@@ -92,6 +94,9 @@ while [ ! -z "$1" ]; do
     shift
     nuttx="$1"
     ;;
+  -p )
+    PRINTLISTONLY=1
```

Comment:
Should we use lower case?

Reply:
why? the majority of variables in this script are with upper case.